List of attributes is out of date

mimckitt opened the pull request:
The list of attributes supported is out of date in this doc. Needs to be updated. Here is the up-to-date list of attributes available: https://docs.microsoft.com/en-us/azure/active-directory/connect/active-directory-aadconnect-version-history

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mimckitt committed:
List of attributes is out of date

The list of attributes supported is out of date. Needs to be updated. Here is the up-to-date list of attributes available: https://docs.microsoft.com/en-us/azure/active-directory/connect/active-directory-aadconnect-version-history
